docs/source/features.rst

```diff
@@ -48,6 +48,32 @@ can be accessed under `/apis/swagger/schema/swagger-ui/#/apis/apis_api_history_e
 .. _django-simple-history: https://django-simple-history.readthedocs.io/en/latest/
 
 
+Management Commands
+^^^^^^^^^^^^^^^^^^^
+
+The APIS history plugin is based on the `django-simple-history` package. It comes with 
+management commands provided by `django-simple-history`. Some of the more useful commands are:
```

Comment:
The first sentence repeats what's already been said at the top of the page. More helpful here would be a direct link to django-simple-history's management commands (docs) for further exploration.

Ex.
The APIS history plugin reuses `django-simple-history`'s [management commands]. Some of the more useful ones are:

Reply:
I would not use the term `reuse` for the management commands. The history app does not use those commands, they simply exist, and that only if the `simple_history` app is enabled.